app/controllers/contact_messages_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ContactMessagesController < ApplicationController
+  def new
+    @contact_message = ContactMessage.new
+  end
+
+  def create
+    @contact_message = ContactMessage.new(contact_message_params)
+    if @contact_message.save
+      NewContactMessageMailer.contact_message_notification(
+        @contact_message).deliver_now
+      flash[:success] =
+        "It's good to hear from you. We will respond to your message shortly."
+      redirect_to root_path
+    else
+      flash.now[:error] =
+        "Please review your message, some information is missing."
+      render :new
+    end
+  end
+
+  private
+
+  def contact_message_params
+    params.require(:contact_message).permit(
+      :name,
+      :email,
+      :cell,
+      :message
```

Comment:
I find it easier to compare two lists if they are alphabetized. For instance I have to add this list into a test, if it's alphabetized and I add an item I know where I need to add it, then next time I am not wondering if both lists match.